src/index.html

```diff
@@ -7,6 +7,460 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="catalog__card card"
+        data-qa="card"
+      >
+
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="Display Apple A1419 iMac 27 inches Retina 5K Monoblock(MNED2UA/A)"
+        >
```

Comment:
```suggestion
      <div
        class="catalog__card card"
        data-qa="card"
      >
        <img
          class="card__image"
          src="images/imac.jpeg"
          alt="Display Apple A1419 iMac 27 inches Retina 5K Monoblock(MNED2UA/A)"
        >
```

It's not necessary to add empty line between a parent and a child